tycho-integration-test/src/main.rs

```diff
@@ -669,6 +670,8 @@ async fn process_state(
                     return HashMap::new();
                 }
             };
+            let min_amount_in = BigUint::from(state.levels.levels[0].quantity.ceil() as u128);
+            min_amount = min_amount_in * BigUint::from(10u32).pow(state.base_token.decimals);
```

Comment:
Can we just add a comment in the code saying why we are doing this like you did in your PR description please